models/data_maps.go

```diff
@@ -38,6 +38,15 @@ const (
 	Error = -1
 )
 
+const (
+	// Default value before adding msg_status column.
+	MsgStatusUnmigrated int = iota
```

Comment:
We usually start these with `iota + 1` in most of the rest of the model files, since 0 is the default for an int column when nothing is assigned.